docs/user/usage.rst

```diff
@@ -0,0 +1,255 @@
+Next, connections supply specification for the block of ``nest.Connect(*)`` function.
+The function ``nest.Simulate(*)`` triggers simulation of your constructed network.
+All recording nodes fills a block to collect activities containing neuronal properties, e.g. node ids and positions, and activity events.
+
+|
+|
+|
+|
+|
+
+Explore network activity
+------------------------
+
+.. image:: ../_static/img/screenshots/activity-explorer.png
+  :width: 100%
+
+Network activity is composed of neuronal properties (positions and ids of neurons) and recorded events from recording devices.
+Events can be subdivided in two groups: spike events or analog signals.
+Spike events contains times and ids of the senders emitting events to the recording devices which can be considered as collectors (``spike detector``).
+Analog signals contains continuous quantities from the recording devices aka samplers (``voltmeter`` or ``multimeter``) interrogate their targets at given time intervals.
+Network activity can be explored in a graph or table.
+
+**Activity graph**
+
+.. image:: ../_static/img/screenshots/activity-graph.png
+  :width: 360px
+  :align: left
+
+It displays either a chart graph or animated 3D graph for the spatial network forming layers in topology which neurons has geographical positions.
+The chart graph contains graphical panels organized in vertical stacks.
+Chart panels is introduced specifically to explore network activity with mouse interaction.
+It visualizes scatter graph for spike activity, line graph for analog signals.
+When you add one or more sub-panels for spike activity it shows a histogram graph of spike times or of inter-spike intervals.
+
+|
+|
+
+**Activity table**
+
+.. image:: ../_static/img/screenshots/activity-table.png
+  :width: 320px
+  :align: right
+
+You can go to the table by clicking on :guilabel:`Activity statistics` button in the right side navigation.
+A table shows simple statistics of recorded elements (rows) of a node (population) connected to a recording device.
+
+In spike events, columns shows spike counts, mean and standard deviation of :math:`ISI` (inter-spike interval) as well as :math:`CV_{ISI}` (Coefficient of variation in inter-spike interval).
+
+In analog signals (e.g. membrane potentials), columns shows :math:`\mu` (mean) and :math:`\sigma` (standard deviation) of analog signal values.
+
+
+|
+|
+|
+|
+|
+|
+|
+
+Manage projects
+---------------
+
+.. image:: ../_static/img/gif/manage-projects.gif
+  :width: 320px
+  :align: left
+
+NEST Desktop has project management helping you to organize your networks and network activity.
+If you want to explore activity of the project, you have to start the simulation (See :ref:`Simulation`).
+
+Clicking on a :guilabel:`New project` creates a new project where you can construct network from the scratch (See :ref:`Construction`).
+It is useful to give project a proper name that you can recognize your projects.
```

Comment:
```suggestion
Clicking on :guilabel:`New project` creates a new project where you can construct a network from the scratch (see :ref:`Construction`).
It is useful to give project a proper name so that you can recognize your projects.
```